vuepress/docs/next/docs/curate/bundle-versions-faq.md

```diff
@@ -1,77 +1,58 @@
-As in many other packaging systems, an Entando Bundle supports versioning, allowing developers to create and release improvements of their package over time.
+# Bundle Versions and Updates - FAQ
 
-## How is a bundle version defined?
+## Support
 
-In the repository, the bundle custom resource comes with a set of `tags`, each one representing a version of the bundle.
-Here an example for a CMS quickstart bundle
-```
-apiVersion: entando.org/v1
-kind: EntandoDeBundle
-metadata:
-    name: cms-quickstart-bundle
-spec:
-  details:
-    name: Entando Example CMS Bundle
-    description: Example Bundle containing CMS components for the Quickstart experience
-  tags:
-    - integrity: ''
-      shasum: ''
-      tarball: 'https://github.com/entando/entando-cms-quickstart-bundle.git'
-      version: v0.0.1
-    - integrity: ''
-      shasum: ''
-      tarball: 'https://github.com/entando/entando-cms-quickstart-bundle.git'
-      version: v0.0.2
-```
+### 1. Does the Entando Platform support bundle versioning?
+A bundle is an independent package containing one or more components. 
+As in other packaging systems, the [Entando Component Registry](../compose/ecr-overview.md)()(ECR) supports bundle versioning, allowing developers to create and release improvements of their package over time.
 
-As you can see from the code above, the bundle `cms-quickstart-bundle` has 2 versions available.
-Each version in the `tags` object must correspond a tag in the Git repository provided in the `tarball` field.
+## How-Tos
+### 1. How do I create a new version of a bundle?
+To release new versions of your bundle after changes, edit the `entando.json` with the new version number for the bundle, then pack and publish your images to Docker. Docker will provide tags to update the bundle version. Once you deploy and install the bundle, the new version number will appear in the App Builder and automatically update in your apps. 
 
-## What format should I use to version my bundle?
+Micro frontends and microservices can have their own version numbers, independent of the bundle version, and can be updated in the same way.
 
-Entando requires versions to follow the [semantic versioning 2.0.0](https://semver.org/#semantic-versioning-200), with the possibility to prepend a `v` to the version itself. Some valid bundle versions are:
+### 2. My bundle contains a microservice generated with the Entando Component Generator, does the version of the microservice have to be the same as the bundle version?
 
-- 1.0
-- v0.1.0-alpha
-- 1.0.1-SNAPSHOT
-- 22.109.10234-RC.1
+The version of the microservice - or the Docker image associated with the microservice - isn't bound to the version of the bundle containing the microservice itself. 
 
+This gives the bundle developer complete control over the bundle release process, especially in those situations where the bundle contains more components and even more microservices.
```

Comment:
i think "complete control over the bundle release process" is broader than this and entails more than versioning; reword slightly to scale back / refine? like complete control over the scope of the release, but not the *process*; the user is never modifying the process created by entando wrt bundle release

"...contains more components and even more MS" ?? this is a bit confusing and i think misleading - the number of microservices doesn't have to be greater than the number of components for targeted releases to come in handy. i think your point is that the more components the more opportunity for MS so the greater the benefit of being able to release on the MS level instead of the bundle level. that kind of granularity accommodates and tracks minor or specific changes without touching the bundle as a whole

Reply:
it's more like why introduce components when versioning applies to MS and bundles? without bridging the gap that components are likely to require MS, thus increasing the number of MS, it's kind of incongruous to bring components into the equation. versioning doesn't apply to components in general, right? -> "...the bundle contains many microservices" would kill my comment

Reply:
You can version ms and mfe. I think you can also version platform components but that's not going to show up in the bundle on Entando??  Used components because that encompasses any/all parts. It doesn't necessarily refer to platform components in general use.